firebase-database/src/androidTest/java/com/google/firebase/database/QueryTest.java

```diff
@@ -3401,6 +3406,101 @@ public void onComplete(DatabaseError error, DatabaseReference ref) {
     IntegrationTestHelpers.waitFor(semaphore);
   }
 
+  @Test
+  public void emptyQueryGet() throws DatabaseException, InterruptedException {
+    DatabaseReference node = IntegrationTestHelpers.getRandomNode();
+    DatabaseConfig cfg = IntegrationTestHelpers.newTestConfig();
+    final Semaphore semaphore = new Semaphore(0);
+    node.get()
```

Comment:
FWIW, you can just do:

```
try {
DataSnapshot snap = node.get().get();
fail(...);
} catch (ExcecutionException e) {
...
}
```

Optional... especially since I just remembered that this might not work since it may block the UI thread that is running the test.